index.html

```diff
@@ -16,7 +16,67 @@
 </head>
 <body>
   <header>
+
+    <div class="header-div">
+      <div class="head-box-text">Code Journal </div>
+    </div>
   </header>
+
+
+
+
+
+  <div class="container">
+
+
+      <div class="form-div" data-view="entry-form">
+
+        <h2 class="form-h2 margin-left-form">New Entry</h2>
```

Comment:
You should put this `h2` into a `row` element with a `column-full` div as its child 
```
row
  column-full
    h2
```
That will likely remove the need for the `margin-left-form` class here